docs/03-computation/02-configuration.mdx

```diff
@@ -59,25 +70,31 @@ Selectors can be set in the PCE using [PUT /configure/selectors](https://stage.c
 
 :::
 
-Sensitive data you capture on a regular base can often be complex to categorize.
+<Legislation>
+Seeting the selectors is important in order to reply to any `TRANSPARENCY.DATA-CATEGORIES` Privacy Request, and comply with several regulations (GDPR Art. 30.1.c, GDPR Art. 15.1.b, CCPA 1798.100.1, CCPA 1798.110.1.1, CCPA 1798.115.1.1, 1798.115.1.2, CCPA 1798.115.1.3). They are also used to indetify particular user data relevant for a particular request and execute operations over the data storage.
+</Legislation>
+
+Sensitive data you capture can often be complex to categorize.
 
 The blindnet devkit gives you a clear [set of data categories](https://github.com/blindnet-io/product-management/blob/main/refs/schemas/priv/dictionary/data-categories/en.data-categories.json) you can use to qualify collected sensitive data for efficient privacy management.
 
-A selector can be defined as a subcategory of any of these data categories, using [dot notation](https://github.com/blindnet-io/product-management/blob/main/refs/schemas/priv/RFC-PRIV.md#term-dot-notation), to qualify [any captured data](https://github.com/blindnet-io/product-management/blob/main/refs/schemas/priv/RFC-PRIV.md#data-capture-fragments).
+A selector, corresponding to a particular field in your databse, can be defined as a subcategory of any of these data categories. This is done using [dot notation](https://github.com/blindnet-io/product-management/blob/main/refs/schemas/priv/RFC-PRIV.md#term-dot-notation).
 
 For example, you can define a `FINANCIAL.BANK-ACCOUNT.IBAN`, as a subcategory of the predefined `FINANCIAL.BANK-ACCOUNT` category, to identify and retrieve an IBAN number collected for regular payment.
 
 <!-- TODO: link to the data capture section instead once it's done -->
 
 _Refer to the definition of [Data Capture Fragments in PRIV](https://github.com/blindnet-io/product-management/blob/main/refs/schemas/priv/RFC-PRIV.md#data-capture-fragments) for more details._
 
+If you are using an automatic data cataloging tool, you may want to consider our [mappings to vocabularies used by major software vendors](https://github.com/blindnet-io/product-management/blob/main/refs/schemas/compatibility.md).
+
 :::caution
 
 Each selector is defined by its full dot notation (`{data_category}.{subcategory}`).
 
 Using just `{subcategory}` will lead to an error.
 
-`{subcategory}` can however be optional, as any data category can be used as a selector.
+`{subcategory}` is however optional, as any data category can be used as a selector.
 
 :::
```

Comment:
In that case, the data category already exists and there is no need to define it. This section can be deleted.

Reply:
True. Deleted